tools/openblas_support.py

```diff
@@ -237,12 +237,12 @@ def get_members():
 
 def make_init(dirname):
     '''
-    Create a _distributor_init.py file for OpenBlas
+    Create a _distributor_init_local.py file for OpenBlas
 
     Obsoleted by the use of delvewheel in wheel building, which
     adds an equivalent snippet to numpy/__init__.py, but still useful in CI
     '''
-    with open(os.path.join(dirname, '_distributor_init.py'), 'w') as fid:
+    with open(os.path.join(dirname, '_distributor_init_local.py'), 'w') as fid:
```

Comment:
I suggest reverting the changes to this file. Until we get rid of this script completely, it's better not to change it.